src/components/bodyImage.tsx

```diff
@@ -22,13 +22,12 @@ const styles = css`
 `;
 
 const imgStyles = (width: number, height: number): SerializedStyles => css`
-    height: calc(100vw * ${height / width});
+    height: ${100 * height / width}%;
```

Comment:
This percentage will be relative to the parent if said parent has a `height` set. Because here the parent _doesn't_ have a height set explicitly, this actually looks ok because I believe it uses the aspect ratio of the image itself to set the height.

Since this is the case you might wonder _why_ we set the `height` at all, since it's set automatically for us by the image. We do it because we're worried about what happens _before_ the image loads. Images typically load in later than the text of the articles, particularly on slow connections. This causes two problems:

1. A caption that doesn't refer to anything visible floating in the middle of a block of text

2. Reflow (the article jumping up and down) as the images load in

We solve this by setting the height of the image tag explicitly, so that it retains a placeholder slot in the page while we wait for the image to load in. That way it makes it more obvious that a caption refers to an image that just hasn't loaded yet, _and_ the images loading in don't cause the page to jump about.

| No Height | Height Set |
| - | - |
| <img width="540" alt="caption" src="https://user-images.githubusercontent.com/53781962/90277446-8d2bbc80-de5d-11ea-9ecd-5df4f5778760.png"> | <img width="542" alt="placeholder" src="https://user-images.githubusercontent.com/53781962/90277491-a16fb980-de5d-11ea-8340-2b60cca2a670.png"> |

In summary, percentages are problematic (something we also need to address in `image-rendering`). I think you may have to do something like this here:

```suggestion
    height: calc((100vw - ${remSpace[4]}) * ${height / width});
```

It's a bit brittle. I think we may need something more robust longer term - again something for the `image-rendering` project to solve.

Reply:
thanks for that explanation Jamie, makes it much clearer, will update 👍